modules/creating-an-infra-node.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="creating-an-infra-node_{context}"]
+= Creating an infra node
+
+[IMPORTANT]
+====
+This procedure applies to user-provisioned infrastructure environments only.
+
+See Creating infrastructure machine sets for installer-provisioned infrastructure
+environments or for any cluster where the master nodes are managed by the
+machine API.
+====
+
+Requirements of the cluster dictate that infra node(s) be provisioned. The
+installer only provides provisions for master and worker nodes. Worker nodes can
+be designated as infra nodes or app nodes through labeling.
+
+.Procedure
+
+. Add a label to the worker node(s) that you want to act as app node(s):
```

Comment:
Slightly confusing that the procedure is titled creating infra nodes, but then this step is labeling a worker node as an app node. Should these be separate procedures, or the title updated to include app node?

Also, no (s) again. I think making them singular should be sufficient here. I won't point out any additional instances in the PR.